.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # java-vertexai has maintainers
-/java-vertexai   @googleapis/vertex-java-sdk @googleapis/cloud-java-team-teamsync
+/java-vertexai/   @googleapis/vertex-java-sdk @googleapis/cloud-java-team-teamsync
 
 *   @googleapis/cloud-java-team-teamsync
```

Comment:
Reading https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file.

Does the `* @googleapis/cloud-java-team-teamsync` needs to be at the top so that the later match takes precedence over global owner?

Reply:
Fixed